docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -219,6 +219,18 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                     only logs for which all conditions are satisfied.
                                     Example: <Filter1> <Filter2> <Filter3> --all -> result
                                     is <Filter1> AND <Filter2> AND <Filter3>. Default is all=true.
+  -c, --conf-names <arg>            Filter applications based on certain Spark properties that were
```

Comment:
sorry just thinking of this now, I"m wondering if we shouldn't rename this to filter-conf because --conf-names could be read as configs for the tool and we don't really need "names" plural since its one at a time.

Reply:
Should we rename it to spark-property ? so that it would be easy for user to mention the spark properties for which the event logs needs to be filtered.

Reply:
yes spark-property is fine